### DISCUSSION TITLE
What does external stock location mean?

### DISCUSSION THREAD

**Discussion by Century1026** [inventree/InvenTree#7268] [Q&A] [answered]
What's "external" functionality?

**Comment by SchrodingersGat** [comment, marked as answer]
An "external" stock location is intended to indicate that the location exists "outside" the normal factory location, and thus the stock contained therein may not be readily available for access.

I'll raise an issue on this as I realise that it is not documented at all.

**Reply by Century1026** [reply]
Alright, I get it, thanks.
Yeah, compared with "structral", it could be more understandable if additional document is provided.
![image](https://github.com/inventree/InvenTree/assets/106154819/5bd1acc4-1138-449d-9cbe-0d2f6fa8e977)

**Reply by turist79** [reply]
Probably adding an option to assign customer company to the stock location if external would improve usability for those, who has FSLs (forward stock locations) located on customer`s premises for maintenance tasks for example.

**Reply by SchrodingersGat** [reply]
Allocated a 'company' reference field to the `StockLocation` model would be a reasonable suggestion. Mostly, it would be blank, but it could be associated with a customer, or even a supplier, etc.
